vms/readme.rst

```diff
@@ -0,0 +1,4 @@
+Title
+-------
+
+Read me.
```

Comment:
```suggestion
```
I don't understand what that file is for. If there is no particular reason, you can remove it.